Evaluation/Input/Content/References.md

```diff
@@ -0,0 +1,144 @@
+
+**Stevens 1998** Stevens, R. E., Limsakun, T., Evans, G., & Mason, J. D. H. (1998). Controlled, multidose, pharmacokinetic evaluation of two extended‐release carbamazepine formulations (Carbatrol and Tegretol‐XR). *Journal of pharmaceutical sciences*, *87*(12), 1531-1534.
+
+**Strandjord 1975** Strandjord, R. E., & Johannessen, S. I. (1975). A preliminary study of serum carbamazepine levels in healthy subjects and in patients with epilepsy. In: *Clinical pharmacology of anti-epileptic drugs* (pp. 181-188). Springer, Berlin, Heidelberg.
+
+**Sumi 1987** Sumi, M., Watari, N., Umezawa, O., & Kaneniwa, N. (1987). Pharmacokinetic study of carbamazepine and its epoxide metabolite in humans. *Journal of pharmacobio-dynamics*, *10*(11), 652-661.
+
+**Terhaag 1978** Terhaag, B., Richter, K., & Diettrich, H. (1978). Concentration behavior of carbamazepine in bile and plasma of man. *International journal of clinical pharmacology and biopharmacy*, *16*(12), 607-609.
+
+**Tomson 1983** Tomson, T., Tybring, G., & Bertilsson, L. (1983). Single‐dose kinetics and metabolism of carbamazepine‐10, 11‐epoxide. *Clinical Pharmacology & Therapeutics*, *33*(1), 58-65.
+
+**US Patent Application - US 2009/0169619 A1**. United States Patent Application 2009, Publication no.: US 2009/0169619 A1. https://patentimages.storage.googleapis.com/66/d4/30/f3588f44ab2b6f/US20090169619A1.pdf, accessed on 12-01-2022.
+
+**US Patent Application - US 2014/0302138 A1**. United States Patent Application 2014, Publication no.: US 2014/0302138 A1. https://patentimages.storage.googleapis.com/57/d9/18/0d8cbfa046681d/US20140302138A1.pdf, accessed on 12-01-2022.
+
+**Vinçon 1987** Vinçon, G., Albin, H., Demotes-Mainard, F., Guyot, M., Bistue, C., & Loiseau, P. (1987). Effects of josamycin on carbamazepine kinetics. *European journal of clinical pharmacology*, *32*(3), 321-323.
+
+**Wada 1978** Wada, J. A., Troupin, A. S., Friel, P., Remick, R., Leal, K., & Pearmain, J. (1978). Pharmacokinetic comparison of tablet and suspension dosage forms of carbamazepine. *Epilepsia*, *19*(3), 251-255.
+
+**Williamson 2016** Williamson, B., Lorbeer, M., Mitchell, M. D., Brayman, T. G., & Riley, R. J. (2016). Evaluation of a novel PXR‐knockout in HepaRG™ cells. *Pharmacology research & perspectives*, *4*(5), e00264.
+
+**Willmann 2007** Willmann, S., Höhn, K., Edginton, A., Sevestre, M., Solodenko, J., Weiss, W., ... & Schmitt, W. (2007). Development of a physiology-based whole-body population model for assessing the influence of individual variability on the pharmacokinetics of drugs. *Journal of pharmacokinetics and pharmacodynamics*, *34*(3), 401-431.
+
+**Wong 1983** Wong, Y. Y., Ludden, T. M., & Bell, R. D. (1983). Effect of erythromycin on carbamazepine kinetics. *Clinical Pharmacology & Therapeutics*, *33*(4), 460-464.
+
+
+
```

Comment:
I cannot find the following references used in the model:
![grafik](https://user-images.githubusercontent.com/25061876/167376474-00502b0c-5b8e-4f24-8c81-b2cb97f2ea27.png)